docs/src/js/redirect.js

```diff
@@ -0,0 +1,4 @@
+// Fore info see issue: https://github.com/sublimelsp/LSP/issues/2284#issuecomment-1606014439
+if (window.location.toString() === "https://lsp.readthedocs.io/en/latest/") {
+	window.location.href = 'https://lsp.readthedocs.io/en/stable/'
+}
```

Comment:
What are your thoughts regarding my comment https://github.com/sublimelsp/LSP/issues/2284#issuecomment-1606018873 ?

That documentation doesn't indicate anywhere that it's for ST3 so I don't feel like it's worth making it more accessible. I'd rather redirect to ST4 documentation.